components/overflow-group/overflow-group-mixin.js

```diff
@@ -28,8 +29,7 @@ export const OverflowGroupMixin = superclass => class extends LocalizeCoreElemen
 	static get properties() {
 		return {
 			/**
-			 * Use predefined classes on slot elements to set min and max slotted items to show
-			 * @type {boolean}
+			 * @ignore
 			 */
 			autoShow: {
```

Comment:
Not (at least initially) supporting autoShow in filter-overflow-group, so moving documentation to just be in `overflow-group`